src/utils/account-with-lockup.js

```diff
@@ -128,7 +128,7 @@ async function getAccountBalance() {
             : totalBalance.sub(BN.max(unreleasedAmount, LOCKUP_MIN_BALANCE))
 
         const lockedAmount = totalBalance.sub(ownersBalance)
-        const liquidOwnersBalance = BN.max(new BN(0), ownersBalance.sub(stakedBalance))
+        const liquidOwnersBalance = BN.min(ownersBalance, new BN(lockupBalance.total))
```

Comment:
I think this fixes something from before, but what about staking rewards earned by the lockup?

Reply:
@mattlockyer staking rewards are part of `stakedBalance` above fetched from staking pool directly (and so are counted in `ownersBalance`).